### ISSUE TITLE
Command and Control improvement

### ISSUE THREAD

**Issue by JDownward** [Savage-Game-Design/Mike-Force#23]
**Is your feature request related to a problem? Please describe.**

Im frustrated that the only way to communicate to air and acav is through side, and the fact coordination is very difficult due to the size of mike team

**Describe the solution you'd like**
to have Dynamic squads inside the teams.

**Describe alternatives you've considered**
in the H menu, have subsection teams to join inside of the main team. 

**Additional context**
To make a example,
Press "H"
Click the team icon "mike force"
then click on the "mike force" team icon
then it has a list of people in each squad and you can join them (all pre-named squads to keep with the theme "MikeForce-1")

**Comment by SyniRon** [comment]
Yes please, 100% this. Especially bad if you use an addon like DUI becasue the huge squad goes off the right of your screen.

**Comment by JDownward** [comment]
this also means we can up the player count aswell

**Comment by JDownward** [comment]
also maybe make it so green hornets have squads that can only use certain aircraft, so dedicated cas/transport

**Comment by Spoffy** [comment]
Thanks for submitting this suggestion. This is surprisingly hard to do, but definitely something we want to look into!
